RadixWallet/Clients/EntitiesVisibilityClient/EntitiesVisibilityClient+Test.swift

```diff
@@ -0,0 +1,23 @@
+extension DependencyValues {
+	public var entitiesVisibilityClient: EntitiesVisibilityClient {
```

Comment:
Do we really need another client for this?

Reply:
We discussed merging PersonasClient and AccountsClients before, at least that is what I suggest, which under the hood (in the Live Impl) can share the same impl of a lot of the functions, or most APIs work with `some EntityProtocol` instead of concrete types. 

And then these new functions would be simple `hideEntity` instead of `hideAccount` and `hidePersona`. 

But I think it can be done as a tech debt item after this PR. i.e. pragmatic fastest way forward is to merge this as is, and then you merge them all into `EntitiesClient`?

Reply:
Yes, this Client is mostly needed since we kinda need a merge of PersonasClient and AccountsClients, and as @CyonAlexRDX suggested we could probably have EntitiesClient, but I'll leave it for another PR.